index.html

```diff
@@ -183,5 +181,30 @@ <h4>LET'S CONNECT</h4>
                 </ul>
                 </nav>
         </section>
+        <footer class="contact">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested 
+                in developing, a feature that you need built 
+                or a project that needs coding. I’d love to 
+                help with it 
+            </p>
+            <form action="https://formspree.io/f/xrgdbdnb"
+            method="POST">
+                <ul class="data-input">
+                    <li>
+                        <input type="text" name="name" placeholder="Enter your name..." required>
+                    </li>
+                    <li>
+                        <input type="email" name="email" placeholder="Write your E-mail" required>
+                    </li>
+                    <li>
+                        <textarea cols="25" rows="6"  name="message" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] would you please add an attribute `maxlength="500"` to your messege  input for validation reasons

Reply:
Added maxlength attribute to message